packages/cli/src/utils/sandbox.ts

```
@@ -1400,11 +1536,37 @@ export async function start_sandbox(
+        );
+        portForwardingResult = await setupPortForwardingPodmanMacOS(
+          portsToForward,
+          SSH_TUNNEL_POLL_TIMEOUT_MS,
+        );
+        portsToForward.forEach((p) => podmanMacOSPortsForwarded.add(p));
+      }
+    }
+
+    // expose env-specified ports on the sandbox (skip if using SSH tunnels on Podman macOS)
+    if (!isPodmanMacOS) {
+      ports().forEach((p) => args.push('--publish', `${p}:${p}`));
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)